docs/Getting Started/Understanding Permissions/index.md

```diff
@@ -233,6 +315,13 @@ sdlManager.getPermissionManager().removeListener(listenerId);
 ```
 !@
 
+@![javascript]
+```js
+sdlManager.getPermissionManager().removeListener(listenerUuid);
+```
+!@
+
+@![iOS,android,javaSE,javaEE]
```

Comment:
The below `Additional HMI State Information` should at least be present, and I think the audio streaming state is important as well. If a developer built a music streaming app or a nav app with WebEngine projection, for example, this would be important information.